bundles/org.openhab.core/src/test/java/org/openhab/core/auth/client/oauth2/AccessTokenResponseTest.java

```diff
@@ -76,6 +76,6 @@ public void testTokenNotExpiredEvenWithBuffer() {
         // token has a life time of 60 seconds
         token.setExpiresIn(60);
 
-        assertTrue(!token.isExpired(Instant.now(), 10), "Token should have been expired due to buffer");
+        assertFalse(token.isExpired(Instant.now(), 10), "Token should have been expired due to buffer");
```

Comment:
Isn't the test and the message a contradiction? We test `isExpired == false` and the failure message says the token should have expired?

Reply:
Fixed the text. I think it was copied from `testTokenExpiredDueToBuffer` to `testTokenNotExpiredEvenWithBuffer`